src/graphorm/index.ts

```diff
@@ -120,6 +144,11 @@ const obj = new GraphORM({
       'private',
       'type',
       'slug',
+      {
+        column: `"contentMeta"->'translate_title'->'translations'`,
+        columnAs: 'i18nTitle',
+        isJson: true,
+      },
```

Comment:
We want to load i18n data all the time so that we can always translate original key to translated one. 

In before implementation I was loading it from new `i18n` column which was a bit smaller but `contentMeta` is big as is so I added support to load `jsonb` nested columns from graphorm.

Now I can just request translated title and then pass it to `createI18nField` so it does the rest.

Reply:
In the future I'd love to fetch only the desired language and only if needed. let's log it as a point for after the experiment. Again my concern is that the translations are heavy and sending them every time over the network is expensive

Reply:
we only send it internally from db to app, we don't send it to user

Reply:
I agree with experiment succeeds we can see to only load specific translation key internally as well

Reply:
i know. still heavy and inefficient since most of the time you'll need only one language and most users will have english as default